Server/src/index.ts

```diff
@@ -76,346 +76,355 @@ export class CitrineOSServer {
+      initSwagger(this._config, this._server);
     }
   }
 
-  private registerAjv () {
+  private registerAjv() {
     // todo type schema instead of any
     const fastifySchemaCompiler: FastifySchemaCompiler<any> = (
       routeSchema: FastifyRouteSchemaDef<any>
     ) => {
-      return this._ajv?.compile(routeSchema.schema) as FastifyValidationResult
-    }
-    this._server.setValidatorCompiler(fastifySchemaCompiler)
+      return this._ajv?.compile(routeSchema.schema) as FastifyValidationResult;
+    };
+    this._server.setValidatorCompiler(fastifySchemaCompiler);
   }
 
-  private initNetworkConnection () {
+  private initNetworkConnection() {
     this._authenticator = new Authenticator(
       this.cacheService?.cache as ICache,
-      new sequelize.LocationRepository(this._config, this._logger),
-      new sequelize.DeviceModelRepository(this._config, this._logger),
-      this._logger
-    )
+      new sequelize.LocationRepository(),
+      new sequelize.DeviceModelRepository(),
+      this.loggerService?.logger
+    );
 
     const router = new MessageRouterImpl(
       this._config,
       this.cacheService?.cache as ICache,
       this._createSender(),
       this._createHandler(),
       async (identifier: string, message: string) => false,
-      this._logger,
+      this.loggerService?.logger,
       this._ajv
-    )
+    );
 
     this._networkConnection = new WebsocketNetworkConnection(
       this._config,
       this.cacheService?.cache as ICache,
       this._authenticator,
       router,
-      this._logger
-    )
+      this.loggerService?.logger
+    );
 
-    this.host = this.configService?.systemConfig.centralSystem.host
-    this.port = this.configService?.systemConfig.centralSystem.port
+    this.host = this.configService?.systemConfig.centralSystem.host;
+    this.port = this.configService?.systemConfig.centralSystem.port;
   }
 
-  private initAllModules () {
-    [
-      this.getModuleConfig(EventGroup.Certificates),
-      this.getModuleConfig(EventGroup.Configuration),
-      this.getModuleConfig(EventGroup.EVDriver),
-      this.getModuleConfig(EventGroup.Monitoring),
-      this.getModuleConfig(EventGroup.Reporting),
-      this.getModuleConfig(EventGroup.SmartCharging),
-      this.getModuleConfig(EventGroup.Transactions)
-    ].forEach((moduleConfig) => {
-      this.initModule(moduleConfig)
-    })
+  private initAllModules() {
+    // [
+    //   this.getModuleConfig(EventGroup.Certificates),
+    //   this.getModuleConfig(EventGroup.Configuration),
+    //   this.getModuleConfig(EventGroup.EVDriver),
+    //   this.getModuleConfig(EventGroup.Monitoring),
+    //   this.getModuleConfig(EventGroup.Reporting),
+    //   this.getModuleConfig(EventGroup.SmartCharging),
+    //   this.getModuleConfig(EventGroup.Transactions),
+    // ].forEach((moduleConfig) => {
+      // this.initModule(moduleConfig);
+    // });
+    this.modules.push(this.transactionsModule as BaseModule);
```

Comment:
temporarily only initializing transactions module to because havent refactored the rest